### PR TITLE
Feature/habit details

### PR DESCRIPTION
**Pull request by GregVaz**
## Explain why this change is being made.
As a registered user with active habits, I want to see the details of a specific habit

## Explain how this is accomplished.
Once a habit is created, you can see all the registered information for the habit

## How do you manually test this?
Given the user is logged in, user can write the url '/habits',  then click on 'Add new habit', fill all the form, an once the habit is created you will be redirected to the details page of the habit

### Screenshots
image

### CODE REVIEW

**Review by jvargas98 — APPROVED** [review]

**Review by LoginALee — APPROVED** [review]
Good Job Goyo ! :taco:

File: app/views/habits/show.html.haml [hunk 1]
```diff
@@ -1,5 +1,11 @@
...
-%p= @habit.start_date
-%p= @habit.end_date
-%p= @habit.status
+.card.mt-4.p-3
+  .card-body
+    %h4.card-title= "Habit: #{@habit.name}"
+    %h6.card-subtitle.mb-2.text-muted= @habit.description
+    .row.mt-4
+      %p.col-6.text-center= "Start: #{@habit.start_date.to_s(:long)}"
+      %p.col-6.text-center= "End: #{@habit.end_date.to_s(:long)}"
```
Comment (OmauryMarmolejo) [inline app/views/habits/show.html.haml]: Do you think we can move these conversions to another place?

**Review by OmauryMarmolejo — COMMENTED** [review]
Great job Gregorio. I left a comment.

**Review by ese-varo — APPROVED** [review]
Good job! :rocket: :guitar:

File: app/views/habits/show.html.haml (hunk 1 above)
Comment (GregVaz) [reply app/views/habits/show.html.haml]: Yeah we can move the responsibility through the model, I already solve this :D

File: app/models/habit.rb [hunk 2]
```diff
@@ -1,4 +1,11 @@
 class Habit < ApplicationRecord
   belongs_to :user
   validates :name, :description, :start_date, :end_date, presence: true
+
+  def start_date_format
+    self.start_date.to_s(:long) 
+  end
```
Comment (OmauryMarmolejo) [inline app/models/habit.rb:7]: Please add a blank line between the two methods

**Review by OmauryMarmolejo — APPROVED** [review]

**Pull request merged**